plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/wrapper/LibvirtPrepareStorageClientCommandWrapper.java

```diff
@@ -31,19 +29,21 @@
 import com.cloud.resource.CommandWrapper;
 import com.cloud.resource.ResourceWrapper;
 import com.cloud.utils.Ternary;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 @ResourceWrapper(handles = PrepareStorageClientCommand.class)
 public class LibvirtPrepareStorageClientCommandWrapper extends CommandWrapper<PrepareStorageClientCommand, Answer, LibvirtComputingResource> {
 
-    private static final Logger s_logger = Logger.getLogger(LibvirtPrepareStorageClientCommandWrapper.class);
+    protected Logger LOGGER = LogManager.getLogger(getClass());
```

Comment:
same here (static upper of instance lowercase)

Reply:
CommandWrapper already has a logger defined which conflicts with this.

Reply:
than why create this one extra?